src/mermaidChartAuthenticationProvider.ts

```diff
@@ -207,9 +205,7 @@ export class MermaidChartAuthenticationProvider
     scopes: readonly string[]
   ) => PromiseAdapter<Uri, string> =
     (scopes) => async (uri, resolve, reject) => {
-      await this.mcAPI.handleAuthorizationResponse(
-        new URLSearchParams(uri.query)
-      );
+      await this.mcAPI.handleAuthorizationResponse(uri.query);
```

Comment:
```suggestion
      await this.mcAPI.handleAuthorizationResponse(`?${uri.query}`);
```

This function currently only works with absolute URLs.

https://github.com/Mermaid-Chart/plugins/pull/5 will add support for relative PRs too, so all we need to do is add a `?` to turn this into a URL with a query string.